DeepSpeech.ipynb

```diff
@@ -148,7 +148,16 @@
    },
    "outputs": [],
    "source": [
-    "dropout_rate = 0.05  # TODO: Validate this is a reasonable value"
+    "dropout_rate = 0.05  # TODO: Validate this is a reasonable value\n",
+    "\n",
+    "# This global placeholder will be used for all dropout definitions\n",
+    "dropout_rate_placeholder = tf.placeholder(tf.float32)\n",
+    "\n",
+    "# The feed_dict used for training employs the given dropout_rate\n",
+    "feed_dict_train = { dropout_rate_placeholder: dropout_rate }\n",
+    "\n",
+    "# While the feed_dict used for validation, test and train progress reporting employs zero dropout\n",
+    "feed_dict = { dropout_rate_placeholder: 0.0 }"
```

Comment:
As we talk about a single scalar value, **I vote for keeping it** - at least for now. If this is really getting a problem, it could be fixed easily.